mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -546,7 +554,12 @@ def __call__(self, datasets):
         # alignment, i.e. the SL ROIs cover roughly the same area
         queryengines = self._get_trained_queryengines(
             datasets, params.queryengine, params.radius, params.ref_ds)
-
+        # For surface nodes to voxels queryengines, roi_seed hardly makes sense
+        if isinstance(queryengines[params.ref_ds], SurfaceVerticesQueryEngine):
+            self.force_roi_seed = False
```

Comment:
There may be future QueryEngines for which force_roi_seed does, or does not, make sense. Can we make this an attribute or method in the QueryEngine? For example a property  .can_force_roi_seed which is True in the base class, but False in `SurfaceVerticesQueryEngine`. What do you think?